public/index.js

```diff
@@ -38,12 +38,36 @@ function volumeDown() {
   }
 }
 
+// positions of insectNet and waterGun
+let insectNetActive = false;
+let waterGunActive = false;
+
 function toggleInsectNet() {
-  const icon = document.querySelector(".tools0");
-  if (icon.classList.contains("insectNet-active")) {
-      icon.classList.remove("insectNet-active");
+  waterGunActive = false;
+  insectNetActive = !insectNetActive;
+  updateToolIcons();
+}
+
+function toggleWaterGun() {
+  insectNetActive = false;
+  waterGunActive = !waterGunActive;
+  updateToolIcons();
+}
+
+function updateToolIcons() {
+  const insectNetIcon = document.querySelector(".tools0");
+  const waterGunIcon = document.querySelector(".tools1");
```

Comment:
```suggestion
  const insectNetIcon = document.querySelector("img.insect-net");
  // TODO: 水鉄砲の機能が実装できたらコメントアウトをやめる
  // const waterGunIcon = document.querySelector("img.water-gun");
```

可読性向上の為、命名は説明的にしましょう。
未使用のロジックは不要なエラーを発生させるため、コメントアウトなどで回避しましょう。